proto/iotics/api/meta.proto

```diff
@@ -26,7 +26,10 @@ service MetaAPI {
   // SparqlQuery performs a SPARQL 1.1 query against the Federated Knowledge Graph of the Iotics network to which this
   // host belongs. The result is returned as a sequence of chunks. Note that:
   // - Result chunks MIGHT arrive out of order and it is the client's responsibility to re-assemble them.
-  // - This RPC is currently in alpha!
+  // - This RPC is currently in beta, it means:
+  //      - the logic should remain unchanged,
+  //      - the rpc call should remain unchanged,
+  //      - the service can be interrupted without notice.
```

Comment:
Just to double check what the intended meaning is: Do you mean it might be turned on/off and thus return `UNIMPLEMENTED`? (Is it worth saying instead in that case that it's an optional feature which might not be enabled on all hosts?)

Reply:
I would prefer not going into the details there until the service will be promoted to stable.